_data/Nis/Nis.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Nishant Gautam 
+institution:  Visvesvaraya Technological University(VTU)
+profile_pic: Nis.png
+quote: Make it or break it , curse it or rejoice it.The life of a developer is a never ending learning Path.
```

Comment:
Quote must **not** be longer than 100 characters, yours is **101** characters.

Reply:
Done can you check it now